algebra/src/fft/domain/test.rs

```diff
@@ -39,16 +42,16 @@ fn fft_composition() {
 
     let rng = &mut rand::thread_rng();
 
-    test_fft_composition::<Bls12_381, _>(rng);
+    test_fft_composition::<MNT6, _>(rng);
 }
 
 #[test]
 fn parallel_fft_consistency() {
-    fn test_consistency<E: PairingEngine, R: rand::Rng>(rng: &mut R) {
+    fn test_consistency<E: PairingEngine, R: Rng>(rng: &mut R) {
         let worker = Worker::new();
 
         for _ in 0..5 {
-            for log_d in 0..10 {
+            for log_d in 0..18 {
```

Comment:
I expect that this number of constraints is addressed to the low two-adicity of the MNT4 base field?